src/cms/adobe-stock-manage.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Using Adobe Stock Images
+---
+
+[Adobe Stock][adobe-stock] images can be used in place of uploading your own image content. One
+common use case is, when creating a new page, it is desirable to upload and place
+image content.
+
+### Access the Adobe Stock Search Grid
+
+The Adobe Stock search panel is accessible when [**Adding** or **Editing** a
+**Page**]({% link cms/page-add.md %}), [**Category**]({% link catalog/category-create.md %})
+or when [inserting images via the **Content Editor**]({% link cms/editor-insert-image.md %}).
+
+To access the Adobe Stock search panel when adding a new page or editing an
+existing page:
+
+1. From the main menu on the left, click **Content** and then **Pages**
+2. Click either **Add a New Page** or, in the grid of page results under the
+   **Action** column, click **Select** and then **Edit**
+3. Expand the **Content** section and:
+    - If you have WYSIWYG mode disabled, click **Insert Image...**
+    - If you have WYSIWYG mode enabled, click **Show/Hide Editor** and then
+      click **Insert Image...**
+4. Click the **Search Adobe Stock** button
+
+To access the Adobe Stock search panel when adding a new category or editing an
+existing category:
+
+1. From the main menu on the left, click **Catalog** and then **Categories**
+2. Click either **Add Root Category**, **Add Subcategory** or, in the list of categories
+   on the left, click an existing category
+3. Expand the **Content** section and under **Category Image** click **Select
+   from Gallery**
+4. Click the **Search Adobe Stock** button
+
+To access the Adobe Stock search panel from anywhere the WYSIWYG Editor is
+available:
+
+1. Click **Show/Hide Editor**
+2. Click **Insert Image...**
+3. Click the **Search Adobe Stock** button
+
+![Adobe Stock Search Results]({% link images/images/adobe-stock/search-grid.png %}){: .zoom}
+
+### Filter and Search for Adobe Stock Assets
+
+The [Adobe Stock Search Grid][access-search] provides querying and filtering functionality
+to help you find the perfect image for your Magento stores.
+
+By default, the search results shown are from an Adobe Stock curated gallery of
+a few hundred results. As soon as you apply your own keyword search, you will be
+searching the millions of assets available via Adobe Stock.
+
+#### Search for Adobe Stock Assets by Keywords
+
+1. [Access the Adobe Stock Search grid][access-search]
+2. Enter your keyword search into the **Search by keyword** input field in the
+   top-left and click the magnifying glass or press **Enter**
+
+![Adobe Stock Search Results for the "mango" keyword]({% link images/images/adobe-stock/mango.png %}){: .zoom}
+
+#### Filter Adobe Stock Assets
+
+1. [Run a keyword search for Adobe Stock assets][search-by-keywords]
+2. Click the **Filters** button
+3. You have several filters available to you to further refine your search
+   results:
+
+    |Filter|Description|
```

Comment:
I think we can also add another column for the default value which is selected in the filter i.e safe search is enabled by default